_projects/_template.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Example Astropy
+desc: This is the first idea for astropy
+# add a description of your project
+requirements:
+ - The proposal has to contain a clear concept.
```

Comment:
I don't understand what this section means.  Requirements *for the student*?  Like "knows some astronomy"?  Or requirements for the project?